### ISSUE TITLE
NVDA doesn't read parent div if It gains focus from shift-tabbing of a child element

### ISSUE THREAD

**Issue by JoeyLevesque** [nvaccess/nvda#10136]
### Steps to reproduce:
SEE BELOW

### Actual behavior:
SEE BELOW

### Expected behavior:
SEE BELOW

#### NVDA installed/portable/running from source:
NVDA installed

#### NVDA version:
Most recent

#### Windows version:
Windows 7
#### Name and version of other software in use when reproducing the issue:
Chrome/Firefox
#### Other information about your system:
Nothing particular
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

Source code here :

    <button>Only there to help you focus the child element </button>
    <div class="container" tabindex="0">
       <a href="#" class="child" aria-label="Child 1" tabindex="0">Child 1</a>
       <a href="#" class="child" aria-label="Child 2" tabindex="0">Child 2</a>
    </div>

JSFiddle : https://jsfiddle.net/Lsz318fb/

The problem arise if I tab from the top of the page until I reach the first child (Child 1 in this case) then if I shift-tab to the div container I would expect NVDA to read all the div (As it does from tabbing from the button to the div), but right now it reads nothing.

Is this a known bug of NDVA ? Is there a solution ?

Current behavior :focus button, tab until you reach the div, tab again. shift-tab. NVDA reads nothing

Expected behavior : focus button, tab until you reach the div, tab again. shift-tab. NVDA should read Child 1 Child 2

**Comment by Adriani90** [comment]
cc: @feerrenrut, @leonardder

**Comment by kaashan** [comment]
Seems like a duplicate of https://github.com/nvaccess/nvda/issues/6606

**Comment by Adriani90** [comment]
Closing as duplicate of #6606.
